openml-python-common/src/main/java/com/feedzai/openml/python/ClassificationPythonModel.java

```diff
@@ -121,11 +123,12 @@ public ClassificationPythonModel(final JepInstance jepInstance,
                                      final String id,
                                      final String classifyFunctionName,
                                      final String getClassDistributionFunctionName) {
-
+        final int targetIndex = schema.getTargetIndex()
```

Comment:
Spent a couple hours around this and IMO it's a bit premature to add support for dataset schemas with no target variable, since we do not have an example or plan of how that look like. Moreover, I think it would even imply changing the "python API" underneath this provider, since the classify method, which now returns the name of the predicted class, would have to return some "normal/abnormal" class, or the index itself (or even other possibility), which in my option is proof for speculation. I think that we should wait until we have an example of an Anomaly Detection algorithm that we wish to be supported by this provider.